embeddings/model/lightning_module/lightning_module.py

```diff
@@ -20,6 +21,7 @@ class LightningModule(pl.LightningModule, abc.ABC, Generic[Model]):
     def __init__(
```

Comment:
I would remove any defaults from model/task classes

Reply:
we have defaults for most of the other models/embeddings defined in our library; do you want to remove all of them or just in the lightning module? it could be done in other PR if you think about the former

Reply:
I think that defaults for model/task should only be stored in Config classes. Because It can lead to some unexpected behaviour.